packages/demux/src/AloxideActionHandler.ts

```diff
@@ -37,13 +42,110 @@ export class AloxideActionHandler extends AbstractActionHandler {
     super(handlerVersions, options);
     if (options) {
       this.indexStateModelName = options.indexStateModelName;
+
+      // add initial handlers
+      const handlers = Array.isArray(options.handlers) ? options.handlers : [];
+
+      for (const { handler, actionName } of handlers) {
+        this.addHandler(handler, actionName);
+      }
     }
   }
 
   getIndexStateModelName() {
     return this.indexStateModelName || `DemuxIndexState_${this.bcName.replace(/\W+/, '_')}`;
   }
 
+  /**
+   * @override
+   * @param candidateType   The incoming action's type
+   * @param subscribedType  The type the Updater of Effect is subscribed to
+   * @param _payload        The payload of the incoming Action.
+   */
+  matchActionType(candidateType, subscribedType, _payload?): boolean {
+    if (subscribedType === '*') {
```

Comment:
I think it's better to allow user to subscribe to action of specific contract account. Somethings like `eosio::*` or `governance::*`